docker/install/ubuntu_install_onnx.sh

```diff
@@ -28,7 +28,7 @@ set -o pipefail
 # to onnx>=1.9, onnxoptimizer should also be installed.
 pip3 install \
     onnx==1.10.2 \
-    onnxruntime==1.9.0 \
+    onnxruntime==1.11.0 \
```

Comment:
Yeah you need to manually update CI containers. For now, just @pytest.skip and give the reason. I'll DM you to deal with upgrading CI image.

Reply:
Undid the change, turns out we still rely on 1.9.0 elsewhere in addition to TVM CI so modified the operator import to cover the older version of the op, left TODOs.